proto/zitadel/oidc/v2alpha/authorization.proto

```diff
@@ -0,0 +1,117 @@
+syntax = "proto3";
+
+package zitadel.oidc.v2alpha;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+option go_package = "github.com/zitadel/zitadel/pkg/grpc/oidc/v2alpha;oidc";
+
+message AuthRequest{
+  option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
+    external_docs: {
+      url: "https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest";
+      description: "Find out more about OIDC Auth Request parameters";
+    }
+  };
+  
+  string id = 1 [
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "ID of the authorization request";
+    }
+  ];
+
+  google.protobuf.Timestamp creation_date = 2 [
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "Time when the auth request was created";
+    }
+  ];
+
+  string application_id = 3 [
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "OIDC client ID of the application that created the auth request";
+    }
+  ];
```

Comment:
the application_id probably originates from the current domain.AuthRequest which can be either the OIDC client_id or SAML entity_id
as this service is OIDC only i think we should go with client_id

Reply:
```suggestion
  string client_id = 3 [
    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
      description: "OIDC client ID of the application that created the auth request";
    }
  ];
```